src/pypulseq/Sequence/block.py

```diff
@@ -254,7 +255,7 @@ def set_block(self, block_index: int, *args: Union[SimpleNamespace, float]) -> N
                     if prev_type == 't':
                         last = 0
                     elif prev_type == 'g':
-                        last = prev_lib['data'][5]
+                        last = prev_lib['data'][2]  # v150
```

Comment:
I would either be more precise and document what was changed. e.g. "`v1.5.0: changed from ['data'][5] to ['data'][2]`", or remove the comment. 

I prefer option 1, but highly recommend a uniform style for all these comments e.g. `v1.5.0: <text>,` which allows us to remove these more easily in the future.